rubocop/lib/rubocop.yml

```diff
@@ -10,7 +10,7 @@ Layout/AlignParameters:
   Enabled: true
 
 Layout/MultilineMethodCallIndentation:
```

Comment:
We need a descriptive comment, which prevents us from re-enabling this (as we did)